src/nreplClient.ts

```diff
@@ -1,45 +1,11 @@
 'use strict';
 
 import * as net from 'net';
-import {Buffer} from 'buffer';
+import { Buffer } from 'buffer';
 
 import * as bencodeUtil from './bencodeUtil';
 
-interface nREPLCompleteMessage {
-    op: string;
-    symbol: string;
-    ns?: string
-}
-
-interface nREPLInfoMessage {
-    op: string;
-    symbol: string;
-    ns: string;
-}
-
-interface nREPLEvalMessage {
-    op: string;
-    file: string;
-    'file-path'?: string,
-    session: string;
-}
-
-interface nREPLSingleEvalMessage {
-    op: string;
-    code: string;
-    session: string;
-}
-
-interface nREPLStacktraceMessage {
-    op: string;
-    session: string;
-}
-
-interface nREPLCloneMessage {
-    op: string;
-}
-
-interface nREPLCloseMessage {
+interface nreplMessage {
```

Comment:
Ah sorry. I think that wasn't even supposed to be commited. I do that to make testing with diferent operations easier. I'm going to revert that.